### PR TITLE
added information about acceleration

### LINKED ISSUE #22885: Explanation of X, y, z coordinates of DeviceMotionEvent.acceleration and DeviceMotionEvent.accelerationIncludingGravity is either completely wrong or extremely confusing (or both)
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/DeviceMotionEvent/acceleration

### What specific section or headline is this issue about?

_No response_

### What information was incorrect, unhelpful, or incomplete?

It says:

> x
> Represents the acceleration upon the x axis which is the **west to east axis**
> y
> Represents the acceleration upon the y axis which is the **south to north axis**
> 
> z
> Represents the acceleration upon the z axis which is the **down to up axis**

Now, the definitions of "west to east" and "south to north" axes are completely unambiguous (unless you are on one of the poles), and crystal clear. And it's **definitely not true** that `x` and `y` correspond to west to east and north to south.

"Down to up" on its own would be potentially ambiguous, but given the other two are east to west and south to north, one can only reasonably assume that **"up to down" is referring to the surface of the earth**. And of course, that is **also not true**.

By  that description, one would unambiguously understand (wrongly) that the coordinates are **absolute and in no way influenced by the orientation of the device**. One can only infer that the system has already done the necessary calculations behind the scene to compute those values (which by the way is only possible by using also the compass as well as the accelerometer; and also I think it would involve some heuristics, not entirely sure about that).

I couldn't believe it but it is so crystal clear and unambiguously stated, that I assumed it had to be true. But of course my gut instinct was right 🤦

### What did you expect to see?

From some quick experiments, I think the reality is:

x: corresponds to the x axis of the html document (not sure if with the same sign or opposite)
y: corresponds to the y axis of the html document (not sure if with the same sign or opposite)
z: corresponds to the remaining perpendicular axis, so one that either points into the screen or out of the screen.

Which of course makes way more sense 🤦

And this is true both for `acceleration` and `accelerationIncludingGravity`

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/devicemotionevent/acceleration`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/DeviceMotionEvent/acceleration
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/devicemotionevent/acceleration/index.md
* Last commit: https://github.com/mdn/content/commit/1511e914c6b1ce6f88056bfefd48a6aa585cebce
* Document last modified: 2022-09-09T05:10:34.000Z

</details>

### PR DESCRIPTION
**Pull request by advitiya7**
### Description

the information about the X,Y,Z were not correct added the correct information with a little explanation beforehand i.e. how to imagine the axes

### Related issues and pull requests

 #22885

### CODE REVIEW

**Comment by advitiya7** [comment]
@wbamberg i have added the necessary changes please check
thanks

File: files/en-us/web/api/devicemotionevent/acceleration/index.md [hunk 1]
```diff
@@ -20,14 +20,19 @@ the gravity force, in contrast to {{DOMxRef("DeviceMotionEvent.accelerationInclu
 ## Value
 
 The `acceleration` property is an object providing information about
-acceleration on three axis. Each axis is represented with its own property:
+acceleration on three axis. There are a couple of things that we need to make sure we understand before we can start using these events: which end is up, and what are the axes we're going to use?
```
Comment (dawei-wang) [inline files/en-us/web/api/devicemotionevent/acceleration/index.md:23]:
```suggestion
acceleration on three axes. There are a couple of things that we need to make sure we understand before we can start using these events: which end is up, and what are the axes we're going to use?
```

**Review by dawei-wang — COMMENTED** [review]
minor typo

**Comment by advitiya7** [comment]
> minor typo

Thanks for pointing that out, i have corrected the typo

**Comment by advitiya7** [comment]
@wbamberg any follow ups on this?

**Comment by sideshowbarker** [comment]
@rakuco If you might be able to make some time to do a technical review of this small fix, that’d be great

**Comment by wbamberg** [comment]
Thank you Mike! I'd really appreciate some expert guidance here.

From a docs-organization point of view, I wanted to point out that we have some conceptual discussion of coordinate systems in the guide page at https://developer.mozilla.org/en-US/docs/Web/API/Device_orientation_events/Orientation_and_motion_data_explained.

If possible we should aim to keep reference pages like https://developer.mozilla.org/en-US/docs/Web/API/DeviceMotionEvent/acceleration relatively terse, and refer to the guide for the conceptual part.

**Review by estelle — COMMENTED** [review]
I have some suggestions to make this easier for people (me) who do no know what this means to understand. As I didn't fully understand the content, my suggested edits may not be correct.

**Comment by bsmth** [comment]
Hi @advitiya7, there's a few review comments outstanding if you'd like to take a look. Thanks a lot for the submission :)

**Comment by advitiya7** [comment]
@bsmth sure will do currently i am busy somewhere else i will check it in a few hours if that is okay?

**Comment by bsmth** [comment]
> i will check it in a few hours if that is okay?

There's no rush, I am taking care that this does not become stalled :)

**Review by Josh-Cena — COMMENTED** [review]
The text here is copied from https://webplatform.github.io/docs/tutorials/device_orientation/ and https://www.w3.org/TR/orientation-event/, which is not acceptable. I'm going to close this so as not to encourage behavior like this.

@estelle, maybe you would want to take your suggestions and submit a new PR?

**Pull request closed without merging**